src/main/java/uk/ac/ebi/eva/contigalias/service/AssemblyService.java

```diff
@@ -69,11 +70,11 @@ public Optional<AssemblyEntity> getAssemblyOrFetchByAccession(String accession)
         if (entities.isPresent()) {
             enaDataSource.addENASequenceNamesToAssembly(entities);
             return entities;
+        } else {
+            throw new AssemblyNotFoundException(
+                    "No assembly corresponding to accession " + accession + " found in the database");
         }
-        fetchAndInsertAssembly(accession);
```

Comment:
Since this isn't fetching the assembly anymore I guess the method should be renamed...  maybe it can be combined with the `getAssemblyByAccession` method since the functionality is now quite similar.